friends/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Home from './component/Home';
+
+const baseUrl = 'http://localhost:5000/friends';
+class App extends Component{
+  constructor() {
+    super()
+    this.state = {
+      friends: ''
+    }
+  }
+
+  FetchFriends() {
+    axios.get(`${baseUrl}`)
+    .then(res =>{
+      console.log("======",res);
+      this.setState({
+        friends: res.data
+      })
+    }).catch(err=> console.log("---------",err));
```

Comment:
You could create some error state to allow a message to be displayed should there be an error